shared/editor/components/Styles.ts

```diff
@@ -11,6 +11,112 @@ export type Props = {
   theme: DefaultTheme;
 };
 
+const mathStyle = (props: Props) => `
+/* Based on https://github.com/benrbray/prosemirror-math/blob/master/style/math.css */
+
+/* == Math Nodes ======================================== */
+
+.math-node {
+	min-width: 1em;
+	min-height: 1em;
+	font-size: 0.95em;
+	font-family: "Consolas", "Ubuntu Mono", monospace;
+	cursor: auto;
+}
+
+.math-node.empty-math .math-render::before {
+	content: "(empty math)";
+	color: red;
+}
+
+.math-node .math-render.parse-error::before {
+	content: "(math error)";
+	color: red;
+	cursor: help;
+}
+
+.math-node.ProseMirror-selectednode { outline: none; }
+
+.math-node .math-src {
+	display: none;
+	color: ${props.theme.codeStatement};
+	tab-size: 4;
+}
+
+.math-node.ProseMirror-selectednode .math-src { display: inline; }
+.math-node.ProseMirror-selectednode .math-render { display: none; }
+
+/* -- Inline Math --------------------------------------- */
+
+math-inline {
+  display: inline; white-space: nowrap;
+
+}
+
+math-inline .math-render { 
+	display: inline-block;
+	font-size: 0.85em;
+	cursor:pointer;
+}
+
+math-inline .math-src .ProseMirror {
+	display: inline;
+	/* Necessary to fix FireFox bug with contenteditable, https://bugzilla.mozilla.org/show_bug.cgi?id=1252108 */
+  border-radius: 4px;
+  border: 1px solid ${props.theme.codeBorder};
+  /* background: ${props.theme.codeBackground}; */
+  padding: 3px 4px;
+  margin: 0px 3px;
+  font-family: ${props.theme.fontFamilyMono};
+  font-size: 80%;
+}
+
+math-inline .math-src::after, math-inline .math-src::before {
+	content: "$";
+	color: ${props.theme.codePunctuation};
+}
+
+/* -- Block Math ---------------------------------------- */
+
+math-display { display: block; }
+
+math-display .math-render { display: block; }
+
+math-display.ProseMirror-selectednode {
+  border-radius: 4px;
+  border: 1px solid ${props.theme.codeBorder};
+  background: ${props.theme.codeBackground};
+  padding: 3px 4px;
+  font-family: ${props.theme.fontFamilyMono};
+  font-size: 80%;
+}
+
+math-display .math-src .ProseMirror {
+	width: 100%;
+	display: block;
+}
+
+math-display .math-src::after, math-display .math-src::before {
+	content: "$$";
+	text-align: left;
+	color: ${props.theme.codePunctuation};
+}
```

Comment:
```suggestion
```

Reply:
I see the logic behind them but it doesn't follow how other blocks work in Outline to show this

Reply:
Agreed on consistency with the styling. However, you can also trigger a block by typing $$<space>, but I think its enough to have it on the KeyboardShortcuts component.